invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessUsers/UserAccessSearchResult.js

```diff
@@ -52,9 +53,12 @@ class UserAccessSearchResultItem extends Component {
                 key={result.id}
               >
                 <Image
-                  src={user.avatar}
+                  src={user.links.avatar}
                   avatar
                   className={user.is_current_user ? "" : "rel-ml-1"}
+                  onError={(e) =>
+                    (e.target.src = "/static/images/square-placeholder.png")
```

Comment:
This is not the most orthodox way of doing this ... However, I didn't want to send two `GET` requests to test if the logo was readable or not by the user. 

Please challenge this